File: Backend/MonsterSeeker/MonsterSeeker/Entities/MonsterEntity.cs
namespace MonsterSeeker.Entities
{
    public class MonsterEntity
    {
        public long Id { get; set; } = default!;

        public string Name { get; set; } = default!;

        public string Description { get; set; } = default!;

        public bool Favourite { get; set; } = default!;
    }
}

File: Project/app/src/main/java/com/example/monsterseeker/database/MonsterEntity.kt
package com.example.monsterseeker.database

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class MonsterEntity(
    @PrimaryKey(autoGenerate = true) val id: Long = 0,
    val name: String,
    val description: String,
    val favourite: Boolean
) {
    companion object {

        fun mock() = MonsterEntity(
            id = 1,
            name = "Monster",
            description = "Super scary monster",
            favourite = false
        )
    }
}